src/utils/helpers.js

```diff
@@ -86,4 +88,70 @@ export const bytesToSize = (bytes) => {
   if (bytes == 0) return '0 Byte';
   var i = parseInt(Math.floor(Math.log(bytes) / Math.log(1024)));
   return Math.round(bytes / Math.pow(1024, i), 2) + ' ' + sizes[i];
+}
+
+//Check if a tag is allowed. All the tag are includes which are supported or will be converted to supported tags.
+export const isAllowedTag = (tag, parentTag) => {
+  if(parentTag === 'li' && ( tag === 'ul' || tag === 'li' || tag === 'ol')){
```

Comment:
You might need to relook into the naming conventions here because `ul` and `ol` should be parentTags. Can cause confusion later.

Reply:
can you explain more?

Reply:
You are checking if parentTag === 'li', while parentTags should be 'ul' or 'ol'. I maybe wrong here.